src/s3path.jl

```diff
@@ -322,17 +322,23 @@ function FilePathsBase.sync(f::Function, src::AbstractPath, dst::S3Path; delete=
     end
 end
 
-function Base.readdir(fp::S3Path)
+function Base.readdir(fp::S3Path; join=false)
```

Comment:
Maybe add `sort=true` while you're at it?

```
  │ Julia 1.4
  │
  │  The join and sort keyword arguments require at least Julia 1.4.
```

Reply:
We should also add this to FilePathsBase and test it in the test suite.

https://github.com/rofinn/FilePathsBase.jl/issues/110